commands/set.md

```diff
@@ -24,9 +25,17 @@ Note: Since the `SET` command options can replace `SETNX`, `SETEX`, `PSETEX`, `G
 OK
 127.0.0.1:6379> GET mykey
 "Hello"
-127.0.0.1:6379> 
 127.0.0.1:6379> SET anotherkey "will expire in a minute" EX 60
 OK
+
+127.0.0.1:6379> SET foo "Initial Value"
```

Comment:
The blank line will never be printed in the valkey-cli session.

Either...

* remove this blank line so all examples are in the same valkey-cli session,
* or split the examples into separate valkey-cli session and put a short text for each example.

If you decide to split the example, I mean something like this:

> Basic usage.
>
> ```
> 127.0.0.1:6379> SET mykey "Hello"
> OK
> 127.0.0.1:6379> GET mykey
> "Hello"
> ```
>
> Set a value and an expiry time.
>
> ```
> 127.0.0.1:6379> SET anotherkey "will expire in a minute" EX 60
> OK
> ```
>
> Conditionally set a value.
>
> ```
> 127.0.0.1:6379> SET foo "Initial Value"
> ...
> ```

Reply:
I have split them, thank you!